integrations/agency_bridge.py

```
@@ -0,0 +1,145 @@
+"""
+import json
+import os
+from pathlib import Path
+
+REPO_ROOT = Path(__file__).parent.parent
+API_DIR   = REPO_ROOT / "api"
+
+
+# ── Loaders ────────────────────────────────────────────────────────────────────
+
+def _load(filename: str):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_load`

(ANN202)